qt/widgets/common/inc/MantidQtWidgets/Common/FitScriptGeneratorMockObjects.h

```diff
@@ -77,13 +77,14 @@ class MockFitScriptGeneratorView : public IFitScriptGeneratorView {
   MOCK_CONST_METHOD1(parameterValue, double(std::string const &parameter));
   MOCK_CONST_METHOD1(attributeValue, Mantid::API::IFunction::Attribute(std::string const &attribute));
 
-  MOCK_METHOD2(removeWorkspaceDomain,
-               void(std::string const &workspaceName, MantidQt::MantidWidgets::WorkspaceIndex workspaceIndex));
+  MOCK_METHOD2(renameWorkspace, void(std::string const &workspaceName, std::string const &newName));
```

Comment:
This doesn't need to be done now, but a quick tip for the future. We now have gmock mock methods so this can easily be done as:
`MOCK_METHOD(void, renameWorkspace, (std::string const&, std::string const &, override))`

This avoids us having to add warning suppressions and will also tell you if you're trying to override something that doesn't exist (such as a typo in the method name)